django/db/migrations/questioner.py

```diff
@@ -114,13 +114,14 @@ def _ask_default(self, default=''):
         string) which will be shown to the user and used as the return value
```

Comment:
I see a few other questions that need to be updated in this file. I am unable to comment on those individual lines, so posting them all here:

- `InteractiveMigrationQuestioner.ask_merge`: "Do you want to merge these migration branches?" --> "Should these migration branches be merged?"
- `InteractiveMigrationQuestioner.ask_rename`: "Did you rename %s.%s to %s.%s (a %s) [y/N]?" -> "Was %s.%s renamed to %s.%s (a %s) [y/N]?"
- `InteractiveMigrationQuestioner.ask_rename_model`: "Did you rename the %s.%s model to %s [y/N]?" -> "Was the model %s.%s renamed to %s [y/N]?"
- `MigrationQuestioner.ask_merge`: Optionally consider updating the docstring.

Reply:
Thank you. Should I consider adding [y/N] to the end of "Should these migration branches be merged?". 
So, the result would look like `"Should these migration branches be merged? [y/N]"`

Reply:
I'm guessing that you're referring to `MigrationQuestioner.ask_merge`? I don't think `[y/N]` is necessary there, since it's only in the docstring and not asked as a question to the user interactively.